wordpress-mobile/WordPressMocks
# Update Gutenberg mocks for WPiOS compatibility

This PR updates the existing Gutenberg opt in/out mocks for compatibility with WPiOS and adds a new mock for when Gutenberg is already enabled.

To test:

* WordPress iOS: Follow the test steps in https://github.com/wordpress-mobile/WordPress-iOS/pull/12551 to test these changes in the WPiOS UI tests. Ensure all tests pass with no WireMock warnings/errors.
* WordPress Android: Check out this branch locally. In the WPAndroid project, update `wp.wordpress_mocks_path` in `gradle.properties` to point to your local `WordPressMocks` repo and run the WPAndroid UI tests. Ensure all tests pass with no WireMock warnings/errors.
